test-app/config/ember-try.js

```diff
@@ -8,24 +8,6 @@ module.exports = async function () {
   return {
     usePnpm: true,
     scenarios: [
-      {
-        name: 'ember-lts-4.4',
-        npm: {
-          devDependencies: {
-            'ember-resolver': '11.0.1',
-            'ember-source': '~4.4.0',
-          },
-        },
-      },
-      {
-        name: 'ember-lts-4.8',
-        npm: {
-          devDependencies: {
-            'ember-resolver': '11.0.1',
-            'ember-source': '~4.8.0',
-          },
-        },
-      },
```

Comment:
I had to remove these scenarios because `@glimmer/component@2.0.0` (specified in the addon's `devDependencies`) requires `ember-source>=4.10`.